vendor/modules.txt

```diff
@@ -1,6 +1,6 @@
 # cloud.google.com/go v0.56.0
 cloud.google.com/go/compute/metadata
-# github.com/3scale/3scale-operator v0.2.1-0.20200730110533-c3b57b704d73
+# github.com/3scale/3scale-operator v0.2.1-0.20201120164439-360b1f382716
```

Comment:
this text is only a comment but it looks to be pointing at an older version

Reply:
https://github.com/3scale/3scale-operator/pull/532/commits sha looks to be 2.10

Reply:
This is correct. Commit sha is what matters here.
btw: no idea why its prefixed with "#" :)